new-project-guidelines.md

```diff
@@ -0,0 +1,111 @@
+# New Project Guidelines
+
+This is guidance for volunteers for how to create a new project for the piscine.
+
+You do not need to read or understand this guidance in order to take the piscine as a trainee.
+
+## Goals
+
+The goals of the piscine are:
```

Comment:
https://piscine.codeyourfuture.io/prep/#goals and https://piscine.codeyourfuture.io/prep/#expectations may be useful to reference here. Or maybe just prep?

Reply:
Added both to the intro section! Thanks!